src/frontend/magnify/package.json

```diff
@@ -59,7 +60,9 @@
     "ts-node": "10.8.1",
     "tslib": "2.4.0",
     "typescript": "4.7.3",
-    "typescript-plugin-css-modules": "3.4.0"
+    "typescript-plugin-css-modules": "3.4.0",
+    "formik": "2.2.9",
+    "yup": "0.32.11"
```

Comment:
Please sort devDependencies alphabetically to prevent futur modification when you will install new deps.